Pin chalk dependency to 2.1.0

logan opened the pull request:
Since chalk 2.2.0, chalk declares types in package.json, which
overrides the @types/chalk dependency and requires a default import.

See https://github.com/chalk/chalk/issues/215 for more context.

samccone commented:
Unfortunately due to the current runtime env of this tool within google, I need to be locked on  2.1 for now.

Commit:
Pin chalk dependency to 2.1.0

Since chalk 2.2.0, chalk declares types in package.json, which
overrides the @types/chalk dependency and requires a default import.
Since this technically breaks backwards compatibility, replace the
caret range with a specific version number.

logan commented:
Google, what a silly place.

Here's an alternate approach to resolving the dev-side dilemma.
